### PR TITLE
test(nemesis.py): Adding kill Scylla monkey nemesis

### PR DESCRIPTION
**Pull request by nizarh11**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -525,6 +525,11 @@ def wait_for_old_node_to_removed():
         finally:
             new_node.running_nemesis = None
 
+    def disrupt_kill_scylla(self):
+        self._set_current_disruption('ScyllaKillMonkey %s' % self.target_node)
+        self.target_node.wait_db_up(timeout=400)
```
Comment (roydahan) [inline sdcm/nemesis.py:530]: Why do you need to "wait_db_up" before you killed it?

Comment (roydahan) [reply sdcm/nemesis.py:530]: move it to after the _kill_scylla_daemon

File: test-cases/longevity/longevity-kill-scylla.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+test_duration: 60
```
Comment (roydahan) [inline test-cases/longevity/longevity-kill-scylla.yaml:1]: remove this file from the PR.

File: sdcm/nemesis.py (hunk 1 above)
Comment (nizarh11) [reply sdcm/nemesis.py:530]: kill_scylla_deamon already has it, will remove it

File: test-cases/longevity/longevity-kill-scylla.yaml (hunk 2 above)
Comment (nizarh11) [reply test-cases/longevity/longevity-kill-scylla.yaml:1]: done

**Pull request merged**